### ISSUE TITLE
Main navigation menu fails accessibility check under certain conditions

### ISSUE THREAD

**Issue by andrewleith** [cds-snc/notification-planning#1149]
@andrewleith commented on [Mon Apr 03 2023](https://github.com/cds-snc/notification-planning-core/issues/52)

## Describe the bug 

When the main navigation menu is in "tablet" size, it fails the axe accessibility tool:
- **Ensures ARIA attributes are allowed for an element's role**
- Element Location:
   - #more-menu
```
<div 
   aria-expanded="false" aria-haspopup="true" id="more-menu" 
   data-module="more-menu" data-module-menu-items="proposition-links" 
   data-module-menu-container="more-menu-container" 
   class="flex-grow text-right bg-gray self-stretch justify-end" 
   style="display: flex; width: 60.1562px;">
```

![image.png](https://images.zenhubusercontent.com/558ca3104f7a08fbb7f36ec0/296e89c1-d8b6-4e2c-8146-d9f15161c565)

**Notes:**
This issue does not occur when the menu doesn't collapse.  It also does not occur in mobile mode even though the menu is also collapsed there.

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
See examples in the documentation

(SEV-1 Critical, SEV-2 Major, SEV-3 Minor, SEV-4 Low) 

## Impact

Describe overall user/system impact to consider when prioritizing this issue.

If applicable

Impact on Notify users:

Impact on Recipients:

Impact on Notify team:

**Comment by amazingphilippe** [comment]
Found a related issue. The more menu places links outside of the main navigation landmark. It would be better if the more menu functioned within the main navigation, instead of moving links outside of it.

**Comment by andrewleith** [comment]
- Waiting on @andrewleith to review

**Comment by YedidaZalik** [comment]
@andrewleith to review.

**Comment by YedidaZalik** [comment]
Phil saw Andrew's review. Moving to QA and will QA once released.

**Comment by YedidaZalik** [comment]
Has been released. Andrew to QA.

**Comment by andrewleith** [comment]
Looks good in PROD, moving to done.
